exercises/pythagorean-triplet/metadata.toml

```diff
@@ -1,4 +1,4 @@
 title = "Pythagorean Triplet"
-blurb = "There exists exactly one Pythagorean triplet for which a + b + c = 1000. Find the product a * b * c."
+blurb = "There exists exactly one Pythagorean triplet for which a + b + c = 1000."
```

Comment:
Should this be "Find the triplet." instead?

Reply:
Yes. Updated.